RELEASE.md

```diff
@@ -0,0 +1,109 @@
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.34.0
+
+## New Freatures and Improvements
+
+* Python backend supports 
+  [Custom Metrics](https://github.com/triton-inference-server/python_backend/tree/r23.05#custom-metrics) 
+  allowing users to define and report counters and gauges similar to the 
+  [C API](https://github.com/triton-inference-server/server/blob/r23.05/docs/user_guide/metrics.md#custom-metrics).
+
+* Python Triton Client defines the 
+  [Triton Client Plugin API](https://github.com/triton-inference-server/client/tree/r23.05#python-client-plugin-api-beta) 
+  allowing users to register custom plugins to add or modify request headers. 
+  This feature is in beta and is subject to change in future releases.
+
+* Improved performance of model instance creation/removal. When the model 
+  instance group is the only model configuration change, Triton will update the 
+  model with the number of instances needed rather than reloading the model. 
+  This feature is limited to non-sequence models only. Read more about this 
+  feature 
+  [here](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_management.md#modifying-the-model-repository) 
+  in bullet point four.
+
+* Added new command line option 
+  [`--metrics-address=<address>`](https://github.com/triton-inference-server/server/blob/r23.05/docs/user_guide/metrics.md#metrics) 
+  allowing the metrics server to bind to a different address than the default 
+  `0.0.0.0`.
+
+* Reduced the default number of model load threads from 2*(number of CPU cores) 
+  to 4. This eliminates Triton hitting resource limits on systems with large CPU 
+  core counts. Use the `--model-load-thread-count` command line option to change 
+  this default.
+
+* Added support for 
+  [DLPack Python specification](https://dmlc.github.io/dlpack/latest/python_spec.html) 
+  in 
+  [Python backend](https://github.com/triton-inference-server/python_backend#pb_utilstensorfrom_dlpack---tensor).
+
+* Refer to the 23.05 column of the 
+  [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) 
+  for container image versions on which the 23.05 inference server container is 
+  based.
+
+## Known Issues
+
+* Tensorflow backend no longer supports TensorFlow version 1.
+
+* OpenVINO 2022.1 is used in the OpenVINO backend and the OpenVINO execution 
+  provider for the Onnxruntime Backend. OpenVINO 2022.1 is not officially 
+  supported on Ubuntu 22.04 and should be treated as beta.
+
+* Some systems which implement `malloc()` may not release memory back to the 
```

Comment:
This point was updated this release. See below:

```
Some systems which implement malloc() may not release memory back to the operating system right away causing a false memory leak. This can be mitigated by using a different malloc implementation. Tcmalloc and jemalloc are installed in the Triton container and can be [used by specifying the library in LD_PRELOAD](https://github.com/triton-inference-server/server/blob/r23.05/docs/user_guide/model_management.md). We recommend experimenting with both tcmalloc and jemalloc to determine which one works better for your use case.
```